pkg/storage/unified/search/bleve.go

```diff
@@ -554,6 +551,25 @@ func toBleveSearchRequest(req *resource.ResourceSearchRequest, access authz.Acce
 		searchrequest.Query = bleve.NewConjunctionQuery(queries...) // AND
 	}
 
+	// Can we remove this? Is access ever nil?
```

Comment:
For now we wire [this implementation](https://github.com/grafana/grafana/blob/bc6b7a4df62224890ae419895f0ed78056b6e374/pkg/storage/unified/resource/access.go#L108) in unistore gprc mode